content/guides/code-splitting.md

```diff
@@ -284,7 +286,6 @@ __src/index.js__
   });
 ```
 
-
```

Comment:
Can you leave this new line in -- leaving two before headers helps readability when editing these documents.